cmd/ak/cmd/triggers/create.go

```diff
@@ -8,15 +8,16 @@ import (
 	"github.com/spf13/cobra"
 
 	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/internal/backend/fixtures"
 	"go.autokitteh.dev/autokitteh/internal/kittehs"
 	"go.autokitteh.dev/autokitteh/internal/resolver"
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
-var event, filter, loc, name string
+var event, filter, loc, name, schedule string
 
 var createCmd = common.StandardCommand(&cobra.Command{
-	Use:     "create <--name=...> <--env=...> <--connection=...> <--event=...> <--loc=...>",
+	Use:     `create <--name=...> <--env=...> <--loc=...> { <--connection=...> <--event=...> | <--schedule=...> }`,
```

Comment:
It's a tiny issue, but you shouldn't use `` quotation marks unless there's a specific reasons for it (regex patterns, strings containing "", or multiline strings).

Also, conceptually, the `loc` parameter should come last (after defining the trigger conditions, you define what the trigger will do).